lib/y2_installbase.pm

```diff
@@ -264,18 +291,37 @@ sub save_strace_gdb_output {
 
 sub post_fail_hook {
     my $self = shift;
-    $self->SUPER::post_fail_hook;
-    get_to_console;
-    $self->detect_bsc_1063638;
-    $self->get_ip_address;
-    $self->remount_tmp_if_ro;
-    # Avoid collectin logs twice when investigate_yast2_failure() is inteded to hard-fail
-    $self->save_upload_y2logs unless get_var('ASSERT_Y2LOGS');
-    return if is_caasp;
-    $self->save_system_logs;
-
-    # Collect yast2 installer  strace and gbd debug output if is still running
-    $self->save_strace_gdb_output;
+    if (check_var("REMOTE_CONTROLLER", "vnc")) {
+        wait_screen_change { send_key 'ctrl-alt-shift-x' };
+        $self->save_remote_upload_y2logs unless get_var('ASSERT_Y2LOGS');
+    }
+    if (check_var("REMOTE_CONTROLLER", "ssh")) {
+        my $remote_ip = get_var('TARGET_IP');
+        my $passwd    = get_var('PASSWD');
+        send_key 'alt-r';
+        send_key 'alt_a';
+        script_run("ssh -X root\@$remote_ip");
+        assert_screen "remote-ssh-login";
+        type_string "yes\n";
+        assert_screen 'password-prompt';
+        type_string "$passwd\n";
+        assert_screen "remote-ssh-login-ok";
+        $self->save_remote_upload_y2logs unless get_var('ASSERT_Y2LOGS');
+    }
+    else {
```

Comment:
With current code we will try to execute this part `if (check_var("REMOTE_CONTROLLER", "vnc"))`. Is that intended?

Reply:
i am not sure i get this comment. if "REMOTE_CONTROLLER" is "vnc" is checked above. So if with "this part" you mean the else block, the answer is no.